### DISCUSSION TITLE
Condition-based picklist

### DISCUSSION THREAD

**Discussion by drakub** [YetiForceCompany/YetiForceCRM#14439] [General]
Dear all, I am been fixed in a situation where I would wish to populate a pick list based on a match of the previous two values from a pick list. For instance, I have a pick list called "A", another one called "B" and the last one is "C". I would wish to select a value from pick list "A" which determines the values to show in pick list "B", this works perfectly well from pick list relations.

Now the difficult issue which I need help for is this; I would wish to to filter the values of pick list "C" that are to be shown based on the values of pick list A and pick list B. Something If A.value=country and B.value==district, show counties belonging to the district from the selected country.

Here is the hierarchy
->Country
      -> State
                 ->Municipality
So I would wish to do something like the above.

Anyone with an idea can get me sorted out.

**Comment by mariuszkrzaczkowski** [comment]
https://gitdeveloper.yetiforce.com/index.php?module=PickListDependency&parent=Settings&view=Edit&sourceModule=Products&sourcefield=usageunit&targetfield=subunit

![image](https://user-images.githubusercontent.com/10198654/112105342-911a7c80-8bac-11eb-8f0e-1e1d42c28c13.png)

**Reply by drakub** [reply]
Understand this perfectly works for Parent->Child relationship. But I wanted an environment where I can also do GrandParent->Parent->Child relationship mapping. How do I do this?

**Comment by rioshaz** [comment]
@job-bob-draku i dont think yf has any solution to this ....
